src/dryad.lisp

```diff
@@ -35,7 +35,12 @@
     :accessor dryad-sprouted?
     :initform (make-hash-table :hash-function #'hash-address :test #'address=)
     :type hash-table
-    :documentation "A map ADDRESS -> BOOLEAN which records whether a `BLOSSOM-NODE' has begun participating in matches."))
+    :documentation "A map ADDRESS -> BOOLEAN which records whether a `BLOSSOM-NODE' has begun participating in matches.")
+   (node-type
```

Comment:
This is similar in flavor to this other mechanism: https://github.com/eigenwareqc/artemis/blob/75b2b7794e2c960a6216032d2083edb18d371dc1/src/node.lisp#L170C40-L170C40 . Dunno which is better.

Reply:
yeah i thought about going the generic route but seemed like slightly more work with no more benefit, at least for my use case. also the supervisor also has a node-class for when it needs to spawn a macrovertex so there's a nice parallel between this and that